packages/babel-config/src/web.ts

```diff
@@ -61,12 +67,22 @@ export const getWebSideBabelPlugins = (
             default: 'React',
             path: 'react',
           },
-          {
+          // A project can turn on trusted graphql documents
+          // If projects do not use trusted documents (default)
+          // it auto-imports the gql tag from graphql-tag
+          !useTrustedDocumentsGqlTag && {
             // import gql from 'graphql-tag'
             default: 'gql',
             path: 'graphql-tag',
           },
-        ],
+          // if projects use trusted documents
+          // then it auto-imports the gql function from the generated codegen client preset
+          useTrustedDocumentsGqlTag && {
+            //  import { graphql as gql} from 'src/graphql/gql'
```

Comment:
Is this comment still correct? Or should it be like this now?
```suggestion
            // import { gql } from 'src/graphql/gql'
```